### DISCUSSION TITLE
Monitoramento de aplicações

### DISCUSSION THREAD

**Discussion by nurycaroline** [frontendbr/forum#1935] [Discussão]
Tenho feito alguns teste com o [Sentry](https://sentry.io/welcome/) e o [DataDog](https://www.datadoghq.com). 
Gostaria de saber quais features vocˆs mais utilizam, e se alguma outra ferramenta?

**Comment by felipefialho** [comment]
Usei o [Sentry](https://sentry.io/welcome/) nos últimos 4 anos e minha experiência foi bastante positiva. A ferramenta é bem poderosa e consegue dar uma visão bem ampla de como anda a saúde das aplicações 😁

Além disso tem ótima integração com praticamente qualquer stack (tanto Front-end quanto Back-end) e também integra com outras ferramentas como o Slack, Trello, Jira, entre outras.

O único ponto negativo é quanto ao tamanho do bundle final, que gera pacotes bem grandes, muitas vezes maior do que a própria aplicação, o que pode gerar problemas de performance, [existem várias issues abertas relacionadas a essa questão](https://github.com/getsentry/sentry-javascript/issues/2707), mas segundo eles essa melhoria está no roadmap da versão 6.

**Comment by wilsonneto-swe** [comment]
@nurycaroline, assim como o @felipefialho citou, o Sentry é muito bom, usei em um projeto e gostei bastante... Mas o que eu mais utilizo atualmente é o Azure Application Insights, acho muito massa como ele coleta tudo de telemetria, e a integração entre logs de backend e de frontend me ajuda demais. Consigo isolar os logs a nível de usuário e ver tudo o que aconteceu no front e no back referente aquele usuário, consigo configurar alertas, ver performance de endpoints e dados de memória e cpu da máquina.... Isto me ajuda demais quando preciso analisar algum ticket de chamado, e mesmo para ver a saúde da aplicação tanto no front quanto no back.
